### DISCUSSION TITLE
Livewire misbehaving on production

### DISCUSSION THREAD

**Discussion by Tech-innovata** [livewire/livewire#4767] [1. Help]
I had a project where by on my local machine the livewire is running properly but on production it's doesn't work rather it displays the codes for livewire insteady I don't know what could be the problem I need help

**Comment by zepfietje** [comment]
Hey @Tech-innovata!

Could you perhaps post a snippet or screenshot of what you're getting on production and what you'd expect?

**Comment by joshhanley** [comment]
@Tech-innovata yeah can you please provide more information as requested above? Thanks!
